src/next-app/pages/quote-request.tsx

```diff
@@ -303,22 +334,22 @@ const QuoteRequestPage: NextPage = () => {
             fetch(`${apiUrl}/quotes/quote-requests/bulk_create/`, {
               method: 'POST',
               body: JSON.stringify({
-                'shops': [1, 4],
-                'description': notes,
-                'vehicle_vin': VIN,
-                'vehicle_make': make === 'Other' ? customMake : make,
-                'vehicle_model': model,
-                'vehicle_year': modelYear,
+                shops: shops.map((shop: any) => shop.id.toString()),
+                description: notes,
+                vehicle_vin: VIN,
+                vehicle_make: make === 'Other' ? customMake : make,
+                vehicle_model: model,
+                vehicle_year: modelYear,
               }),
               headers: {
-                'Authorization': `JWT ${authData.access}`,
+                Authorization: `JWT ${authData.access}`,
                 'Content-Type': 'application/json; charset=UTF-8',
               },
             }).then((response) => {
               console.log(response);
               response.json().then((response) => {
                 console.log(response);

```

Comment:
In regards to my previous comment, I think we just have to add a router push here.

Reply:
Oh right, this was implemented by Gilbert, I only added the Select field. But I'll update this. Good catch!